pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.53</version>
+    <version>4.0-beta-5</version>
```

Comment:
I would still like to see `-Duse-jenkins-bom` in the mvn config

Reply:
It’s on by default in the 4.x line afaik

Reply:
neat I must have missed that 😅